kratos/sources/registry_item.cpp

```diff
@@ -0,0 +1,70 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
```

Comment:
Tabs!

Reply:
```suggestion
// Kratos default license: kratos/license.txt
```

Reply:
Done, as well as the similar observations you posted above.